stylesheet.css

```diff
@@ -594,3 +632,553 @@ textarea {
 .project_image:hover {
   transform: scale(1.05); /* increase image size by 10% */
 }
+
+.desktop {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  body {
+    display: flex;
+    flex-direction: column;
+    max-width: 100vw;
+    margin: 0;
+    overflow-x: hidden;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .desktop {
+    display: flex;
+  }
+
+  header {
+    display: flex;
+    flex-direction: row;
+    height: 5vh;
+    background-color: #3c3a39;
+    max-width: 100vw;
+    width: 100vw;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .logo {
+    padding: 0 0 0 5vw;
+  }
+
+  .nav-links {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    width: 30vw;
+    align-items: center;
+    justify-content: space-around;
+    padding: 0 5vw;
+  }
+
+  .nav-li {
+    color: #fff;
+    text-decoration: none;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-size: 0.9375rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.25rem;
+  }
+
+  .nav-li:hover {
+    color: #ff6b00;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 90vh;
+    width: 100vw;
+    background-image: url('./images/background/headline-background-desktop.svg');
+    background-repeat: no-repeat;
+    background-size: 50% 100%;
+  }
```

Comment:
As raised earlier, kindly ensure that the background image is placed properly as we have it in figma. Though, I noticed that it worked fine below the screen size of 905px, I guess the relative unit you used for the `backround-size` is interfering. ℹ️ 

<img width="561" alt="Untitled" src="https://github.com/ommarz/Portfolio/assets/103386183/1b115f57-3ddb-4bc3-81ad-5e86107ee560">